extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -43,14 +44,15 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     /** @private {?Element} The element that is transitioned. */
     this.ampImgEl_ = null;
 
-    /** @private {?float} Percentage between [-50; 50] */
-    this.x_ = null;
-
-    /** @private {?float} Percentage between [-50; 50] */
-    this.y_ = null;
+    /** @private {?{x: float, y: float, zoom: float}} Position to animate to. */
```

Comment:
Please create a typedef so you can write better types for animationState_ or animate_()